hooks/pre_gen_project.py

```diff
@@ -80,3 +80,8 @@
         "Mail Service for sending emails."
     )
     sys.exit(1)
+
+"""
+{{ cookiecutter.update({ "domain_name": cookiecutter.domain_name | trim }) }}
+{{ cookiecutter.update({ "email": cookiecutter.email | trim }) }}
+"""
```

Comment:
That is pretty hacky, but nice trick... Since it's not very obvious what this does, could we:

- Add a comment explaining that it's not just a string, it serves a purpose.
- Add a test for it: generate a project with extra spaces in the domain/email and check that the outpur doesn't include them

Should we move this string more at the top of the file? If we start using these values in the file, I guess they wouldn't be cleaned yet?

Reply:
ok, let's me update this pr.